Tribler/Core/Modules/restapi/display_endpoint.py

```diff
@@ -0,0 +1,89 @@
+import json
+
+from twisted.web import http, resource
+
+
+class DisplayEndpoint(resource.Resource):
+    """
+    This endpoint is responsible for handling requests for the trust display.
```

Comment:
It is already clear that it's an endpoint, maybe change it to "handling requests for the trust display".